src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -142,6 +144,12 @@ public int parseArguments(Parameters params) throws CmdLineException {
   )
   String googProvidesFile = null;
 
+  @Option(
+    name = "--knownClassAliases",
+    usage = "file containing a map of names that we know are aliases"
```

Comment:
It would nice to describe the expected format for this file. Probably the usage string is too short for this, but maybe in a regular comment